CHANGES.md

```diff
@@ -15,6 +15,11 @@
   directory path in the scope of the `DatastoreManager` class.
 - [FIX] Corrected a case where two root nodes with identical revision IDs prevented selection of the
   correct new winning revision.
+- [FIX] Added migration on `Datastore` opening to repair datastores that have documents with
+  multiple identical revision IDs (as caused by issue 329). The migration will delete duplicate
```

Comment:
As a rule we've tended not to mention specific issue numbers in the CHANGES, because the focus is on telling a library consumer what has changed from their perspective. So I'd generally shy away from discussing internals too much here.